deterministic-build.yml

```diff
@@ -0,0 +1,42 @@
+trigger:
+- master
+
+pool:
+  vmImage: windows-latest
+
+steps:
+- checkout: self
+  path: WalletWasabi
+
+- task: UseDotNet@2
+  displayName: 'Install .NET Core 3'
+  inputs:
+    version: 3.1.x
+
+- task: PowerShell@2
+  displayName: 'Build and compare'
+  inputs:
+    targetType: 'inline'
+    script: |
+      # Change git output from stderr to stdout https://github.com/dahlbyk/posh-git/issues/109.
+      $env:GIT_REDIRECT_STDERR = '2>&1'
+      cd ..
+      # Copy the source files to a different location and build from there.
+      robocopy WalletWasabi WalletWasabiClone /COPYALL /S /XD ".git" /NFL /NDL /NJH
+      dotnet run -p WalletWasabi\WalletWasabi.Packager --onlybinaries
```

Comment:
It looks to me that if this command fails then the rest of the commands will still be executed. 

In Bash, people typically use idiom like `command || exit 1`.

Reply:
I tested it if the command is failing powershell will fail so the overall job will fail. 

```
dotnet : Unhandled exception. System.IO.FileNotFoundException: Could not find file 
'D:\a\1\WalletWasabiClone\WalletWasabi.Gui\bin\dist\wassabee_1.1.12_amd64.deb'.
At D:\a\_temp\a9d924d4-c55a-4e35-afd7-b8b981ec75d6.ps1:9 char:1
+ dotnet run -p WalletWasabiClone\WalletWasabi.Packager --cdelivery34
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (Unhandled excep....12_amd64.deb'.:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
##[error]PowerShell exited with code '1'.
Finishing: Build and compare

```

Reply:
If it does not continue after getting non-zero exit code, how come that `if (0 -eq $LASTEXITCODE)` works?

Also locally, when I run this script:

```powershell

echo "Test"

dotnet ----version

echo "Test2"
```

I can see both messages printed. So I'm confused.